index.js

```diff
@@ -38,30 +38,31 @@ export default class Drawer extends Component {
       const height = Number.parseInt(window.getComputedStyle(drawer._targetRef).height, 10);
       if (height) {
         drawer.setState({ memoizedTargetHeight: height });
+    
       }
       oldTargetOnTransitionEnd();
     }
-
+    
     const maxHeight = drawer.state.isOpen ? (drawer.state.memoizedTargetHeight || '100vh') : 0;
-
+    
     const style = {
       ...target.props.style,
       transition: '.25s max-height linear',
       maxHeight,
-      overflow: 'auto'
+      overflow: 'hidden'
```

Comment:
The only issue I see with this (and the reason it was `auto` to begin with) is if the menu contents extends beyond the height of the window (e.g. a phone in landscape mode).